### ISSUE TITLE
Add -e REPLICATION_INFINITE_TIMEOUT=true

### ISSUE THREAD

**Issue by monopollly** [tarantool/docker#14]
Problem with several replication sources.

start 2 servers with 3 replication sources...

```
docker run -p 3301:3301 -e TARANTOOL_REPLICATION_SOURCE=10.0.1.2:3301,10.0.1.2:3302,10.0.1.2:3303
docker run -p 3302:3301-e TARANTOOL_REPLICATION_SOURCE=10.0.1.2:3301,10.0.1.2:3302,10.0.1.2:3303
```

and then got error and ALL servers failed in one time...
its unacceptable
```
ER_CFG: Incorrect value for option 'replication_source': failed to connect to one or more servers
2016-11-07 03:11:33.190 [1] main/101/tarantool-entrypoint.lua F> can't initialize storage: Incorrect value for option 'replication_source': failed to connect to one or more servers
```

snapshot_period: 3600
wal_dir: /var/lib/tarantool
listen: 3301
pid_file: /var/run/tarantool/tarantool.pid
replication_source:
- test:@10.0.1.2:3301
- test:@10.0.1.2:3302
- test:@10.0.1.2:3303
snap_dir: /var/lib/tarantool

could you fix it as soon as possible.
Kostya said about timeouts...

Thank you!

**Comment by racktear** [comment]
@monopollly it should be fixed in Tarantool 1.7. Please try and report here.

**Comment by racktear** [comment]
Closing, as this is not an issue for tarantool 1.7
